File: ClientApp/src/app/objects/character.ts
export class Character {
    id: string;
    name: string;
    imageSrc: string;
    nickname: string;
    description: string;
    shortDescription: string;
    x: number;
    y: number;

    constructor(id: string, name: string, imageSrc: string, nickname: string, description: string, shortDescription: string, x: number, y: number) {
        this.id = id;
        this.name = name;
        this.imageSrc = imageSrc;
        this.nickname = nickname;
        this.description = description;
        this.shortDescription = shortDescription;
        this.x = x;
        this.y = y;
    }
}

File: TreeWikiApi/DAL/Entities/Character.cs
using System;
using System.Collections.Generic;

namespace DAL.Entities
{
    public class Character
    {
        public Guid Id { get; set; }

        public string Name { get; set; }

        public string ImageSrc { get; set; }

        public string Nickname { get; set; }

        public string Description { get; set; }

        public string ShortDescription { get; set; }

        public int X { get; set; }

        public int Y { get; set; }

        public Guid WorkId { get; set; }
    }
}
